src/misc/test/confirm.spec.ts

```diff
@@ -0,0 +1,22 @@
+import { confirm } from "../confirm";
+import { publish } from "../../pubSub/pubSub";
+import { stdToGeneratorPublishedMessage as prefix } from "../helpers";
+
+describe("confirm test", () => {
+  it("should return true when confirmed", async () => {
+    setTimeout(() => {
+      publish(`${prefix}.confirm.ok`);
+    }, 1000);
+
+    let result = await confirm("Confrim?");
+    expect(result).toBeTruthy();
+  });
+  it("should return false when canceled", async () => {
+    setTimeout(() => {
+      publish(`${prefix}.confirm.cancel`);
+    }, 1000);
+
+    let result = await confirm("Confrim?");
+    expect(result).toBeFalsy();
```

Comment:
I'd be stricter here with the expected value too

```suggestion
    expect(result).toBy(false);
```

Reply:
Fixed